inspector/src/components/persistentPopupHostComponent.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+import { PopupComponent } from "./popupComponent";
+import { Nullable } from "babylonjs/types";
+import { GlobalState } from "./globalState";
+import { Observer } from "babylonjs/Misc/observable";
+
+interface IPersistentPopupHostComponentProps {
+    globalState?: GlobalState;
+}
+
+interface IPersistentPopupHostComponentState {
+    renderFunction?: Nullable<() => React.ReactNode>;
+}
+
+/**
+ * This component is a persistent popup host that subscribes to any popup changes 
+ * @param props 
+ */
+export class PersistentPopupHostComponent extends React.Component<IPersistentPopupHostComponentProps, IPersistentPopupHostComponentState> {
+    private _renderFunctionChangeObserver?: Nullable<Observer<Nullable<() => React.ReactNode>>>;

```

Comment:
as this is initialized in the constructor it doesn't need to be optional and nullable. If it is needed, choose one of the two (right now its type is: `null | undefined | () => React.ReactNode`). Null is usually enough, unless there is explicit need for `undefined`.